src/ngrxTemplates/add-user-modal/add-user-modal.component.html

```diff
@@ -1,4 +1,4 @@
-<div class="users-modal">
+<div class="users-modal" *ngIf="!shouldDeleteUser">
```

Comment:
keep file name as generic if using for both add and delete 
don't keep add-user-modal